### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by siddontang**
@queenypingcap  @overvenus 

PTAL @lucab

### CODE REVIEW

**Comment by overvenus** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+let metric_familys = r.gather();
+encoder.encode(&metric_familys, &mut buffer).unwrap();
+
+// Output to the standard output.
+println!("{}", String::from_utf8(buffer).unwrap());
+```
+
+### Next steps
+
++ [API Document](https://docs.rs/prometheus)
```
Comment (lucab) [inline README.md]: This is usually displayed as a badge in the top row: prometheus

**Comment by siddontang** [comment]
PTAL @lucab

**Comment by lucab** [comment]
LGTM too

**Pull request merged**